vlib/v/checker/tests/index_sumtype_interface_struct_params_err.out

```diff
@@ -0,0 +1,34 @@
+vlib/v/checker/tests/index_sumtype_interface_struct_params_err.vv:11:9: warning: pointer indexing is only allowed in `unsafe` blocks
+    9 |     _ = bar[0]
+   10 | 
+   11 |     _ = st1[0]
+      |            ~~~
```

Comment:
This should be an error, not a notice. `mut st1 Struct` does not allow indexing *at all*, even in unsafe.

Reply:
It should be an error, for the same reason that `st3[0]` is.

Reply:
Oh, I see! Thanks!  👍

Reply:
done, thanks for review!